src/routes/basket.ts

```diff
@@ -9,7 +9,7 @@ router
     // .get('', getBaskets)
     // .get('/:id', getBasketById)
     // .post('', checkToken({ secret: `${process.env.JWT_PRIVATE_KEY}`, algorithms: ['HS256']}), addBasket)
-    .patch('/:id', updateBasket)
+    .patch('/:id', checkToken({ secret: `${process.env.JWT_PRIVATE_KEY}`, algorithms: ['HS256']}), updateBasket)
```

Comment:
That's great that you added JWT validation here, but as far as I see it still misses in the basket controller. We have to check if the basket that is being changed "belongs" to the logged user.

Reply:
the best is to move it entirely to routes/me.ts as we will use it only for logged users, and the rest of basket controllers will be attached there